### PR TITLE
fix(dotnet): download and build on windows

### PR DESCRIPTION
**Pull request by sjk07**

### CODE REVIEW

File: template/condo.ps1 [hunk 1]
```diff
@@ -148,7 +148,9 @@ function Write-Info([string] $message) {
 
 function Get-File([string] $url, [string] $path, [int] $retries = 5) {
     try {
+        [Net.ServicePointManager]::SecurityProtocol = "tls12, tls11, tls"
```
Comment (dmccaffery) [inline template/condo.ps1]: can you replace this entire file with the one I gave you? it adds a for-loop to make the retries actually work ;)

**Review by dmccaffery — DISMISSED** [review]
thank you, so much, Microsoft -- you piece of shit

File: template/condo.ps1 (hunk 1 above)
Comment (sjk07) [reply template/condo.ps1]: fiiiiine

**Review by tiffanywang3 — APPROVED** [review]
:rubber-stamp:

**Review by dmccaffery — APPROVED** [review]

**Comment by dmccaffery** [comment]
LGTM! :shipit:  ⚓️

**Pull request merged**